ci/licenses_golden/licenses_flutter

```diff
@@ -2414,6 +2414,41 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
+====================================================================================================
```

Comment:
Please use Copyright 2013 in all places below, then you shouldn't need this

Reply:
done.